Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_3_3.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### JOB - Integrations and Incidents Health Check
+- Added a checks if found failed tasks before running the command 'RestartFailedTasks'.
```

Comment:
- Added a check for failed tasks before running the 'RestartFailedTasks' command.